package.json

```diff
@@ -5,8 +5,8 @@
   "dependencies": {
     "@capacitor/core": "^4.6.1",
     "@capacitor/preferences": "^4.0.2",
-    "@ionic/react": "^7.7.2-dev.11707425083.192d8103",
-    "@ionic/react-router": "^7.7.2-dev.11707425083.192d8103",
+    "@ionic/react": "^8.0.0-rc.1",
+    "@ionic/react-router": "^8.0.0-rc.1",
```

Comment:
Do we have a ticket to come back to update these when v8 comes out?

Reply:
yes we have `FW-5873` for it